selfdrive/car/subaru/values.py

```diff
@@ -171,6 +180,10 @@ def init_make(self, CP: car.CarParams):
       obd_multiplexing=False,
     ),
   ],
+  # We don't get the EPS from non-obd queries on gen2 cars
+  non_essential_ecus={
+    Ecu.eps: [c.value for c in GLOBAL_GEN2]
```

Comment:
```suggestion
    Ecu.eps: list(GLOBAL_GEN2),
```

ugh PyCharm thinks value is a function, you can list it since StrEnum subclasses from str